roles/common/tasks/main.yml

```diff
@@ -1,4 +1,23 @@
 ---
+- name: Remove specified repository from sources list
+  ansible.builtin.apt_repository:
+    repo: 'deb https://oss-binaries.phusionpassenger.com/apt/passenger {{ ansible_distribution_release }} main'
```

Comment:
@kayiwa Looks like you need a sane default for ansible_distribution_release in common.  This is failing CI